device-types/Supermicro/SSG-6029P-E1CR12T.yml

```diff
@@ -0,0 +1,45 @@
+---
+manufacturer: Supermicro
+model: SuperStorage 6029P-E1CR12T
+slug: supermicro-superstorage-6029p-e1cr12t
+part_number: SSG-6029P-E1CR12T
+u_height: 2
+is_full_depth: true
+weight: 27.7
+weight_unit: kg
+airflow: front-to-rear
+comments: '[Technical Specification](https://www.supermicro.com/en/products/system/2u/6029/ssg-6029p-e1cr12t.cfm)'
+console-ports:
+  - name: Serial
+    type: de-9
+power-ports:
+  - name: PSU-1
+    type: iec-60320-c14
+    maximum_draw: 1200
+  - name: PSU-2
+    type: iec-60320-c14
+    maximum_draw: 1200
```

Comment:
Are these PSU's modular? If so, they should be modelled under `module-bays` and the associated PSU modules should be added to the module-types folder if they do not already exist